pam/pam_api.yml

```diff
@@ -274,6 +286,20 @@ components:
       required: true
       schema:
         type: string
+    traceparent:
+      name: traceparent
+      description: Describes the position of the incoming request in its trace graph. Further specified in https://www.w3.org/TR/trace-context
+      in: header
+      required: false
+      schema: 
+        type: string
+    tracestate:
+      name: tracestate
+      description: Extends traceparent with vendor-specific data represented by a set of name/value pairs. Further specified in https://www.w3.org/TR/trace-context
+      in: header
+      required: false
+      schema:
+        type: string
```

Comment:
Did some investigation how to send tracing data, but for plain rpc that we use I couldn't find any other option than to clearly include it in the spec. For grpc on the other hand there are some other mechanism to it as metadata